### DISCUSSION TITLE
How to set cache-config option for the vault transit secret engine

### DISCUSSION THREAD

**Discussion by lawl-dev** [bank-vaults/bank-vaults#3129] [Q&A]
Hey there,
I'm trying to configure the `cache-config` option for the vault transit secret engine. According to [this issues](https://github.com/bank-vaults/bank-vaults/issues/2462) and [this MR](https://github.com/bank-vaults/bank-vaults/pull/2463), the option seems to be supported and implemented.

Could anyone guide me on how to set this option in my YAML file?
Best,
Maxim

**Comment by csatib02** [comment]
Hey @lawl-dev,

Would you be able to share your YAML file?
